node/config/def.go

```diff
@@ -148,6 +148,13 @@ func DefaultBoost() *Boost {
 			MaxConcurrencyStorageCalls: 100,
 			GCInterval:                 lotus_config.Duration(1 * time.Minute),
 		},
+		IndexProvider: lotus_config.IndexProviderConfig{
+			Enable:               false,
```

Comment:
Do we want index announcement to be disabled by default?

Reply:
So, we enabled it by default in Lotus RC but disable it by default in Lotus master. 

I'm doing what Lotus RC does by default and enabling it for now. We can disable later if we need to.